packages/server/test/UserProfile.test.ts

```diff
@@ -29,4 +29,22 @@ describe(`${PATH}`, () => {
     expect(resp.status).toBe(HttpStatus.OK);
     expect(created).toEqual(expect.objectContaining(resp.body));
   });
+
+  it('PUT /', async () => {
+    // Given
+    const user = {
+      name: 'User',
+      profileCompleted: false,
+    };
+    const created = await fixture.db.userModel.create(user);
+    const id = created._id?.toString();
+    const accessToken = accessTokenAsCookie(fixture.app.get(JwtService).sign({ role: Role.User, sub: id }));
+    const reqBody = { name: 'John', surname: 'Doe' };
+
+    // When
+    const resp = await fixture.agent().put(PATH).set('Cookie', [accessToken]).send(reqBody);
+
+    // Then
+    expect(resp.status).toBe(HttpStatus.NO_CONTENT);
+  });
```

Comment:
Fajny byłby dodatkowy `expect`, który sprawdza, czy obiekt faktycznie się zaktualizował, ale jakby Ci to miało zająć dużo czasu to sobie daruj, żeby się udało dziś bez 🚩 XD